music_assistant/server/providers/tidal/helpers.py

```diff
@@ -84,12 +84,14 @@ async def get_artist_albums(session: TidalSession, prov_artist_id: str) -> list[
     """Async wrapper around 3 tidalapi album functions."""
 
     def inner() -> list[TidalAlbum]:
+        try:
+            artist_obj = TidalArtist(session, prov_artist_id)
+        except HTTPError as err:
```

Comment:
uhm what about other HTTP errors ? So e.g. when the server has having other issues etc.
Maybe just catch the 404 (if the provider raises that) and let anything else just bubble up.

Foods for thought is if we should introduce a ProviderTemporaryUnavailable exception if the provider server is temporary down which we can handle. But that is for another (rainy) day.